kasparovd/controllers/common.go

```
@@ -36,6 +36,10 @@ func convertTxDBModelToTxResponse(tx *dbmodels.Transaction) *apimodels.Transacti
 		Inputs:          make([]*apimodels.TransactionInputResponse, len(tx.TransactionInputs)),
 		Outputs:         make([]*apimodels.TransactionOutputResponse, len(tx.TransactionOutputs)),
 		Mass:            tx.Mass,
+		Version:         tx.Version,
+	}
+	if tx.RawTransaction != nil {
```

When will RawTransaction ever be nil? Please return an error if never.